src/epoll_ioctl.c

```diff
@@ -40,14 +40,14 @@ epoll_ioctl(struct tcb *const tcp, const unsigned int code,
 {
 	switch (code) {
 	case EPIOCSPARAMS:
-		tprint_arg_next();
+		tprints_arg_next_name("argp");
 		print_struct_epoll_params(tcp, arg);
 
 		return RVAL_IOCTL_DECODED;
 
 	case EPIOCGPARAMS:
 		if (entering(tcp)) {
-			tprint_arg_next();
+			tprints_arg_next_name("argp");
 			return 0;
 		}
 		print_struct_epoll_params(tcp, arg);
```

Comment:
I suggest moving this `tprints_arg_next_name` from the entering phase to the exiting, i.e. right before `print_struct_epoll_params`.